test/e2e/edu-1990-rjsf-helpers.js

```diff
@@ -1,90 +0,0 @@
-import _ from 'lodash/fp';
```

Comment:
Looks like this was all just copied into `edu-1990-helpers.js`, aye? This diff isn't super helpful. 😒

Reply:
Yeah, it's not. I just renamed this file.